template/config/meta/package.json

```diff
@@ -7,7 +7,12 @@
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
     "test:e2e": "cypress open",
-    "test:unit": "vue-cli-service test:unit"
+    "test:unit": "vue-cli-service test:unit",
+    "storybook:serve": "run-s storybook:tailwind storybook",
+    "storybook:build": "build-storybook -c .storybook -o dist/storybook",
+    "storybook": "start-storybook -s ./public -p 8000 --ci storybook:tailwind",
+    "storybook:tailwind": "npx tailwindcss build src/theme/tailwind.css -o .storybook/tailwind.css"
```

Comment:
Replace all these storybook commands with the version from the reference project. All the commands are necessary:

"build-storybook": "run-s storybook:build storybook:copyres",
"storybook": "run-s storybook:tailwind storybook:serve",
"storybook:build": "build-storybook -c .storybook -o dist/storybook",
"storybook:serve": "start-storybook -s ./public -p 8000 --ci",
"storybook:copyres": "ncp ./public/images dist/storybook/images",
"storybook:tailwind": "npx tailwindcss build src/theme/tailwind.css -o .storybook/tailwind.css",

Reply:
Also, these commands **MUST** to be defined within the storybook mainfest's package section.